setup.py

```diff
@@ -5,10 +5,18 @@
 
 INSTALL_REQUIREMENTS = [
     "Django>=1.11,<2.0",
-    "django-cms>=3.5.0",
+    "django_cms",
```

Comment:
Can this be left as:     "django-cms>=3.5.0", ?

Reply:
Yes, this is now changed.